src/components/Tooltip/Tooltip.js

```diff
@@ -0,0 +1,285 @@
+import React, { useState, useEffect, useRef, Fragment } from 'react'
+import PropTypes from 'prop-types'
+import { DialogOverlay, DialogContent } from '@reach/dialog'
```

Comment:
@bwalker8038 is your modal done? I think this component should probably just use your modal if so